siliconcompiler/flows/asicflow.py

```diff
@@ -52,14 +52,15 @@ def make_docs():
     chip.set('flowarg', 'place_np', n)
     chip.set('flowarg', 'cts_np', n)
     chip.set('flowarg', 'route_np', n)
-    setup_flow(chip)
+    chip.set('flow', 'asicflow')
+    setup(chip)
 
     return chip
 
 ###########################################################################
 # Flowgraph Setup
 ############################################################################
-def setup_flow(chip):
+def setup(chip, flowname='asicflow'):
```

Comment:
Maybe a change for a future PR, but if we make this `flowname` keyword arg a standard for flow setup() functions, we could let the user optionally override it through a kwarg to `load_flow`. Could be a nifty feature.

Reply:
As an example use case, I'm imagining a user writing a target that sets up multiple variations of one flow, e.g.:

```python
chip.set('flowarg', 'proc_np', '1')
chip.load_flow('asicflow', name='serial_asicflow')
chip.set('flowarg', 'proc_np', '10')
chip.load_flow('asicflow', name='parallel_asicflow')
```

Reply:
Interesting idea!